src/app/globals.css

```
@@ -101,39 +140,81 @@ body {
+  border: 2px solid #9370DB; /* Default: pixel-purple */
   border-radius: 4px;
   background-color: white;
+  color: black;
   transition: all 0.2s ease;
 }
 
+.dark .pixel-input {
+  background-color: var(--beige-hex);
+  border-color: var(--accent-pastel-blue-hex);
+  color: var(--dark-brown-hex);
+  placeholder-color: var(--light-brown-hex); /* For dark mode placeholder */
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)